src/wp-admin/edit-tag-form.php

```diff
@@ -14,7 +14,7 @@
 // Back compat hooks
 if ( 'category' == $taxonomy ) {
 	/**
- 	 * Fires before the Edit Category form.
+	  * Fires before the Edit Category form.
```

Comment:
This probably needs manual fixing, there's actually a space before the tab but now it's after the tab. 🤔

Reply:
https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/977